LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2017 Seegno
+Copyright 2019 Uphold, Inc
```

Comment:
Copyright notices typically use either the year of first publication, or a range, such as `2017-2019`. See [this post](https://ben.balter.com/2015/06/03/copyright-notices-for-websites-and-open-source-projects/#the-date) for more details.

By the way, we might need two copyright lines if indeed two entities hold copyright for different time periods.

Finally, a tiny detail, but I believe `Inc.`, being an abbreviation, requires a period at the end.

Reply:
That post adds more questions that answers :laughing: 

Apart from the entities, which I am not sure of, the licence is updated.

Reply:
I can change it to: `Copyright [year project started] - [current year], [project founder] and the [project name] contributors.`?

```
Copyright 2015 - 2019, Rui Marinho and the `koa-requestid` contributors.
```

Reply:
Short answer: yes, that works for our purposes, and nicely sidesteps the company name uncertainty.

Long answer: That really depends on whether the contributors' contracts (at the time of each contribution) specified that their intellectual property is entirely transferred to their employer for work done under those contracts, or merely the copyrights to such work are granted to the employer. The distinction is really too minor for us to bother in this context. A proper assessment would IMO only be justified if we were talking about more sensitive material.

Reply:
I'd prefer to just switch to Uphold here. This was a remnant of a migration that was missed. 

@waldyrious's comments are still valid. I believe it would be appropriate to change it to:

```
Copyright 2015 - 2019, Uphold Inc.
```